docs/UserGuide.md

```diff
@@ -41,25 +41,49 @@ MediBase3 (MB3) is a **desktop app for doctors to manage their patients and appo
+`DOB` | Date of birth (DOB)  of the patient.                                                                       | <ul><li>Must be in the format `YYYY-MM-DD`. <li>Cannot be a date in the future. <ul>                                                                                                                                                                        | <ul><li>:white_check_mark:`2002-12-12`<li>:x:`2002/11/32`</ul>
+`GENDER` | Gender of the patient.                                                                                    | <ul><li>Case-insensitive. <li>Should only be either `M` (Male) or `F` (Female).                                                                                                                                                                             | <ul><li>:white_check_mark:`m`<li>:white_check_mark:`F`<li>:x:`Male`</ul>
+`EMAIL` | Email address of the patient.                                                                              | Must be in the format `prefix@domain`.                                                                                                                                                                                                                      | <ul><li>:white_check_mark:`techraj@gmail.com`<li>:x:`techraj@gmail`</ul>
+`ADDRESS` | Address of the patient.                                                                                   | Both alphanumeric and special characters are allowed.                                                                                                                                                                                                       | :white_check_mark:`Orchard Road, Block 124, #02-01`
+`PHONE_NUMBER` | Phone number of the patient.                                                                             | <ul><li>Should only contain numbers. <li>Spaces and symbols are not allowed</ul>                                                                                                                                                                            | <ul><li>:white_check_mark:`98765432`<li>:x:`+65 9876 5432`</ul>
+
+[Back to Table of Contents](#table-of-contents)
+
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`, `NAME` is a parameter which can be used as `n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `edit NRIC [n/NAME] [i/NRIC] [g/GENDER] [d/DOB] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]` can be used as `edit S1234567A n/John Lim g/M` or as `edit S1234567A g/M`.
```

Comment:
I think for `edit` there could be an issue since we are allowing users to edit their NRIC through `i/NRIC` and hence the initial NRIC must be a premable.